src/rotator_library/AGENTS.md

```
@@ -0,0 +1,95 @@
+#HN|# Rotator Library Knowledge Base
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)